packages/web-components/src/calendar/calendar.styles.ts

```diff
@@ -148,14 +150,59 @@ ${display("inline-block")} :host {
 `.withBehaviors(
   forcedColorsStylesheetBehavior(
       css`
-          .day.selected {
-              color: ${SystemColors.Highlight};
-          }
-
-          .today .date {
-              background: ${SystemColors.Highlight};
-              color: ${SystemColors.HighlightText};
-          }
+        :host {
+          forced-color-adjust: auto;
+          background: ${SystemColors.ButtonFace};
+          outline: 1px solid ${SystemColors.CanvasText};
```

Comment:
Is this adding a border similar to a card? If so, I think we want to have a transparent border in the base styles and then set the color here. Let's try to keep `outline` for focus state only.

Reply:
updated and set a transparent border in default contrast